File: server/java/src/main/java/com/hrms/business/abstracts/EmployerService.java
package com.hrms.business.abstracts;

import com.hrms.core.utilities.results.DataResult;
import com.hrms.core.utilities.results.Result;
import com.hrms.entities.concretes.Candidate;
import com.hrms.entities.concretes.Employer;

import javax.xml.crypto.Data;
import java.util.List;

public interface EmployerService {

    Result add(Employer employer);
    Result update(Employer employer);
    DataResult<List<Employer>> getAll();
    DataResult<Employer> getById(int id);
    Result approveUpdate(int id);
}

File: client/react/hrms/src/services/employerService.js
import axios from "axios";

export default class EmployerService {
  getAll() {
    return axios.get("http://localhost:8080/api/employers/getall");
  }
  getById(id) {
    return axios.get("http://localhost:8080/api/employers/getbyid?id=" + id);
  }
  update(employer) {
    return axios.post("http://localhost:8080/api/employers/update", employer);
  }
}
